src/mui/list/List.js

```diff
@@ -102,10 +106,9 @@ export class List extends Component {
         return this.props.location.pathname;
     }
 
-    refresh = (event) => {
+    refresh = event => {
```

Comment:
now that the action is global, you'd better move that logic to a `RefreshButton` component.